core/chains/solana/fees/fixed_price.go

```diff
@@ -0,0 +1,36 @@
+package fees
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana/config"
+)
+
+var _ Estimator = &fixedPriceEstimator{}
+
+type fixedPriceEstimator struct {
+	cfg config.Config
+}
+
+func NewFixedPriceEstimator(cfg config.Config) (Estimator, error) {
+	if cfg.DefaultComputeUnitPrice() < cfg.MinComputeUnitPrice() || cfg.DefaultComputeUnitPrice() > cfg.MaxComputeUnitPrice() {
+		return nil, fmt.Errorf("default price (%d) is not within the min (%d) and max (%d) price bounds", cfg.DefaultComputeUnitPrice(), cfg.MinComputeUnitPrice(), cfg.MaxComputeUnitPrice())
```

Comment:
nit/ This is a big hard to read.
```suggestion
	default, min, max := cfg.DefaultComputeUnitPrice(), cfg.MinComputeUnitPrice(), cfg.DefaultComputeUnitPrice()
	if default < min || default > max {
		return nil, fmt.Errorf("default price (%d) is not within the min (%d) and max (%d) price bounds", default, min, max)
```